youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -290,6 +290,16 @@ def run(self, information):
 
         try:
             self._downloader.to_screen('[ffmpeg] Destination: ' + new_path)
+
+            if information.get('artist') is not None:
+                more_opts += ['-metadata', 'artist={}'.format(information['artist'])]
+
+            if information.get('title') is not None:
+                more_opts += ['-metadata', 'title={}'.format(information['title'])]
+
+            if information.get('album') is not None:
+                more_opts += ['-metadata', 'album={}'.format(information['album'])]
+
```

Comment:
What's the point of this at all? It's already implemented in `FFmpegMetadataPP`.